api/v4/source/bookmarks.yaml

```diff
@@ -6,7 +6,7 @@
       description: |
         Creates a new channel bookmark for this channel.
 
-        __Minimum server version__: 9.4
+        __Minimum server version__: 9.5
```

Comment:
I don't think we're getting into 9.4 as the rc-1 was cut recently

Reply:
There are other bookmarks docs referencing 9.4 we should look into it before meging the feature

Reply:
Agree, we need to review all references to the MM version before merging